draft-ietf-tls-tls13.md

```diff
@@ -847,7 +849,7 @@ secret keying material. Once the handshake is complete, the peers
 use the established keys to protect application layer traffic.
 
 A failure of the handshake or other protocol error triggers the
-termination of the connection, optionally preceded by an alert message
+termination of the connection, optionally preceded by an Alert message
```

Comment:
I don't think we need to go through here changing all these capitalization. Please fork all of the message type capitalization changes off into a separate PR for separate consideration to the typo and grammar changes. (and do a squash of all the little commits into one or a few)